src/robot/result/model.py

```diff
@@ -37,10 +37,14 @@
 
 from collections import OrderedDict
 from itertools import chain
+from typing import Sequence
 import warnings
 
 from robot import model
 from robot.model import BodyItem, create_fixture, Keywords, Tags, TotalStatisticsBuilder
+from robot.model.modelobject import DataDict
+from robot.model.testcase import TestCases
+from robot.model.testsuite import TestSuites
```

Comment:
All these should be importable directly from `robot.model`. If not, they need to be exposed.